skyportal/tests/frontend/test_reminders.py

```diff
@@ -0,0 +1,192 @@
+import os
+import uuid
+import time
+
+from skyportal.tests import api
+from datetime import date, timedelta, datetime
+from selenium.webdriver.common.keys import Keys
+
+
+def post_and_verify_reminder(endpoint, token):
+    reminder_text = str(uuid.uuid4())
+    next_reminder = datetime.utcnow() + timedelta(seconds=5)
+    reminder_delay = 1
+    number_of_reminders = 2
+    request_data = {
+        'text': reminder_text,
+        'next_reminder': next_reminder.strftime("%Y-%m-%dT%H:%M:%S"),
+        'reminder_delay': reminder_delay,
+        'number_of_reminders': number_of_reminders,
+    }
+
+    status, data = api(
+        'POST',
+        endpoint,
+        data=request_data,
+        token=token,
+    )
+    assert status == 200
+    assert data['status'] == 'success'
+
+    status, data = api('GET', endpoint, token=token)
+    assert status == 200
+    assert data['status'] == 'success'
+    data = data['data']['reminders']
+    assert data[-1]['text'] == reminder_text
+    assert data[-1]['next_reminder'] == next_reminder.strftime("%Y-%m-%dT%H:%M:%S")
+    assert data[-1]['reminder_delay'] == reminder_delay
+    assert data[-1]['number_of_reminders'] == number_of_reminders
+
+    time.sleep(20)  # wait for the reminder to be sent
```

Comment:
even more sleep??

Reply:
I'll try to get rid of it the best I can.

Reply:
It's just that we have the reminder in 5 seconds from now, + the service that iterates over all reminders every 5 seconds. So in the worse case scenario we need to wait around 10 seconds, best case scenario around 5 seconds. So a sleep will be needed. Maybe what I can do is a 5 seconds sleep, and then query the API `n` times to see if it was sent?

Reply:
I would first off make the reminder 1 second instead of 5, and then do a loop with 1 second intervals. That would probably reduce the wait times in most cases.